packages/client/types.tsx

```diff
@@ -106,6 +112,10 @@ interface SearchTracksResultsParams {
     tracks?: TrackMetadata[];
 }
 
+interface MpeRoomParams {
+    id: string;
```

Comment:
I really don't wanna be annoying, but I think the name `id` is not idiomatic to our current conventions
We've always add a prefix such as room or user and we've never used, not saying this a good thing, `id` using small caps
Should this params named `roomID` ?

Reply:
It could be, indeed.
As a lot of things will be renamed or refactored in the next few days, I did not want to waste too much time on them.